content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -286,74 +335,6 @@ Custom user actions are user actions declared and sent manually via the [`addUse
 {{% /tab %}}
```

Comment:
Replace link 4 with /real_user_monitoring/browser/data_collected/?tab=useraction#how-is-the-action-loading-time-calculated

Reply:
Replace link 5 with real_user_monitoring/browser/tracking_user_actions/?tab=npm#custom-user-actions